theano/tensor/tests/test_basic.py

```diff
@@ -73,8 +73,34 @@
 else:
     mode_opt = get_default_mode()
 
-### seed random number generator so that unittests are deterministic ###
+
+# Use a seeded random number generator so that unittests are deterministic
 utt.seed_rng()
+test_rng = numpy.random.RandomState(seed=utt.fetch_seed())
+
+
+# A helpful class to check random values close to the boundaries
+# when designing new tests
+class MockRandomState:
```

Comment:
It helped me detect and fix issues, but it is not a replacement that should be used in general (there are issues with minimum, maximum, for instance, because all values are the same, which would not happen with actual random samples).
So I figured it would be useful to have that code here. Do you have a better idea of where to put it, so it is ready to use when adding new tests?